src/components/SecondaryPanes/Breakpoints/Breakpoint.js

```diff
@@ -39,12 +39,12 @@ type Props = {
   breakpoint: BreakpointType,
   breakpoints: BreakpointsMap,
   selectedSource: ?Source,
-  disableBreakpoint: Function,
+  source: Source,
+  frame: ?Frame,
   enableBreakpoint: Function,
   removeBreakpoint: Function,
-  selectSpecificLocation: Function,
-  source: Source,
-  frame: ?Frame
+  disableBreakpoint: Function,
```

Comment:
we should stop being lazy and using Function. The best would be `typeof actions.disableBreakpoint` where flow does the work for us

Reply:
note `typeof` doesn't work well with thunk actions; but yes otherwise

Reply:
it might still capture arguments, right?

Reply:
yeah it should be fine if you're only interested in the arguments.

Reply:
mmm nope sorry: Flow might prevent you from passing the function, because `typeof thunkAction` returns a function, while the function you pass to your component returns whatever _that_ function returns. This is a case where `typeof` is too precise if you only care about the arguments.
This works for simple actions because what the action creator returns is what the `dispatch(action)` call returns too.

But please check what I'm saying against a real-world use case.